readme.md

```diff
@@ -77,6 +77,85 @@ That's it! Restart Slack and see how well it works.
 
 NB: You'll have to do this every time Slack updates.
 
+# Installing into Franz
+
+If you are running Slack as a service inside Franz, the above instructions will not help you. You need
+to tweak the service definition for Slack to inject the custom CSS.
+
+Find your Franz's service configuration for Slack.
+
+* Windows: ???
+* Mac: `~/Library/Application Support/Franz/recipes/slack`
+* Linux: ???
+
```

Comment:
I would recommend creating a franz-mac.md file for documenting franz install on mac.
leave the notes in the main readme above this, and link to the new md file in the Mac note.

Reply:
Directory for windows would be as follows:
%HOMEDRIVE%%HOMEPATH%\AppData\Roaming\Franz\recipes\slack